src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/resourcecard/ResourceCard.kt

```diff
@@ -16,7 +18,9 @@ import tornadofx.FX.Companion.messages
 
 class ResourceCard(private val resource: Resource) : HBox() {
 
-    val isCurrentResource = SimpleBooleanProperty(false)
+    val isCurrentResourceProperty = SimpleBooleanProperty(false)
+    var primaryColorProperty = SimpleObjectProperty<Color>(Color.ORANGE)
+    var primaryColor: Color by primaryColorProperty
```

Comment:
NEW 4/5/19: To test this property, go to `ResourceGroupCard.kt` and change line 15 to
```
resourcecard(it) {
     primaryColor = Color.(X)
}
```